motionplan/kinematic_test.go

```diff
@@ -389,3 +389,25 @@ IK:
 
 	return solutions, nil
 }
+
+// Test loading model kinematics of the same arm via ModelJSON parsing and URDF parsing and comparing results.
+func TestKinematicsJSONvsURDF(t *testing.T) {
+	numTests := 100
+
+	mJSON, err := frame.ParseModelJSONFile(utils.ResolveFile("components/arm/universalrobots/ur5e.json"), "")
+	test.That(t, err, test.ShouldBeNil)
+	mURDF, err := frame.ParseURDFFile(utils.ResolveFile("referenceframe/testurdf/ur5_viam.urdf"), "")
+	test.That(t, err, test.ShouldBeNil)
+
+	seed := rand.New(rand.NewSource(50))
+	for i := 0; i < numTests; i++ {
```

Comment:
Is the intention to change the seed for each iteration?

Reply:
Not currently. We do get a different random joint configuration for each iteration of the loop, however, but for each run of the test we do get the same joint sequences.

Reply:
In the `GenerateRandomConfiguration` function over in `model.go` (L55), we end up pulling a DoF-length amount of random numbers from the `seed` each time we go through the loop.

Reply:
More generally:

Random number seeds in Go are deterministic, reusable sources of randomness. So if you call:
```
	seed := rand.New(rand.NewSource(50))
	fmt.Println(seed.Int())
	fmt.Println(seed.Int())
	fmt.Println(seed.Int())
```
You'll get three different numbers, but they'll be the same three numbers each time you run the code.

Reply:
thanks!